doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -9,100 +9,92 @@ You can use the process instance migration (PIM) service to create, export and e
+$ curl https://raw.githubusercontent.com/kiegroup/process-migration-service/main/ddl-scripts/postgresql/postgresql-quartz-schema.sql -o postgresql-quartz-schema.sql
+$ curl https://raw.githubusercontent.com/kiegroup/process-migration-service/main/ddl-scripts/postgresql/postgresql-pim-schema.sql -o postgresql-pim-schema.sql
 ----
-. Ensure the relevant authentication method is set for `auth-method`. Valid values are `file`, `jdbc`, and `ldap`. For example:
+
+.. Create the tables:
 +
-.`ldap` method example
+[source]
 ----
-pim:
-  auth-method: ldap
-quakus:
-  security:
-    ldap:
-      dir-context:
-        url: ldap://localhost:10389
-        principal: uid=admin,ou=system
-        password: secret
-      identity-mapping:
-        search-base-dn: ou=users,dc=example,dc=com
-        rdn-identifier: uid
-        attribute-mappings:
-          from: uid
-          filter: uniqueMember
-          filter-base-dn: ou=groups,dc=example,dc=com
+$ psql -U jboss -h localhost -d rhpam7 -f ~/Downloads/postgresql-quartz-schema.sql
+$ psql -U jboss -h localhost -d rhpam7 -f ~/Downloads/postgresql-pim-schema.sql
 ----
+
+. Change directory to the `process-migration` directory.
+
+. Use a text editor to create the `servers.yaml` configuration file with the following content and save in the `process-migration` directory:
 +
-.`file` method example
-+
+[source]
 ----
-pim:
-  auth-method: file
-quarkus:
-  security:
-    users:
-      file:
-        enabled: true
-        plain-text: true
-        users: users.properties
-        roles: roles.properties
+kieservers:
+  - host: http://localhost:8080/kie-server/services/rest/server
+    username: rhpamAdmin
+    password: password1!
 ----
+
+. Use a text editor to create the `datasource.yaml` configuration file with the following content `datasource.yaml` and save in the `process-migration` directory:
 +
-.`jdbc` method example
-+
+[source]
 ----
-pim:
-  auth-method: jdbc
 quarkus:
-  jdbc:
-    principal-query:
-      sql: SELECT u.password, u.role FROM users u WHERE u.username=?
-      clear-password-mapper:
-        enabled: true
-        password-index: 1
-      attribute-mappings:
-        0:
-          index: 2
-          to: groups
+  datasource:
+    db-kind: postgresql
+    jdbc:
+      url: jdbc:postgresql://localhost:5432/rhpam7
+    username: jboss
+    password: jboss
 ----
-. Optional: To configure connections to one or more {KIE_SERVERS}, add the {KIE_SERVERS} to the custom configuration file, for example:
+
+. Rebuild the `quarkus-run.jar` file to include the PostgreSQL driver:
 +
+[source]
 ----
-kieservers:
-  - host: http://kieserver1.example.com:8080/kie-server/services/rest/server
-    username: admin
-    password: admin123
-  - host: https://kieserver2.example.com:8443/kie-server/services/rest/server
-    username: foo
-    password: bar
+$ java -jar -Dquarkus.launch.rebuild=true -Dquarkus.datasource.db-kind=postgresql quarkus-app/quarkus-run.jar
```

Comment:
This command has a similar output to this:
INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 2657ms